easybuild/easyconfigs/v/VAMPIRE-ASM/VAMPIRE-ASM-6.0_makefile.patch

```diff
@@ -0,0 +1,28 @@
+--- vampire-6.0/makefile.orig	2024-04-16 09:59:14.730461427 +0200
```

Comment:
Please add a despription to the patch. See https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/a/AMOS/AMOS-3.1.0_link_correctly_to_DELTAFILER_SHOWCOORDS.patch for an example.

Reply:
Switched to `MakeCp` easyblock (and deleted patch) as suggested by @jfgrimm.